demo/index.html

```diff
@@ -123,7 +123,9 @@ <h3>auro-hyperlink examples</h3>
 
       <demo-snippet>
         <template>
+          <div style="background-color: var(--auro-color-background-darkest); padding: 10px 20px">
```

Comment:
Please remove the padding. The purpose of the demo is to illustrate the element and only the space that it takes up. So all elements in the demo space go edge to edge.